src/mesh/PhoneAPI.cpp

```diff
@@ -253,11 +257,15 @@ size_t PhoneAPI::getFromRadio(uint8_t *buf)
             fromRadioScratch.moduleConfig.which_payload_variant = ModuleConfig_audio_tag;
             fromRadioScratch.moduleConfig.payload_variant.audio = moduleConfig.audio;
             break;
+        case ModuleConfig_remote_hardware_tag:
+            fromRadioScratch.moduleConfig.which_payload_variant = ModuleConfig_remote_hardware_tag;
+            fromRadioScratch.moduleConfig.payload_variant.remote_hardware = moduleConfig.remote_hardware;
+            break;
         }
 
         config_state++;
         // Advance when we have sent all of our ModuleConfig objects
-        if (config_state > ModuleConfig_audio_tag) {
+        if (config_state > (_AdminMessage_ModuleConfigType_MAX + 1)) {
```

Comment:
Oh man, we should have been doing that all along and we missed it!

Reply:
Well it breaks if there's a gap in the numbering.